Code/Tools/ProjectManager/Source/NewProjectSettingsScreen.h

```diff
@@ -55,12 +62,16 @@ namespace O3DE::ProjectManager
         QButtonGroup* m_projectTemplateButtonGroup;
         QLabel* m_templateDisplayName;
         QLabel* m_templateSummary;
+        QPushButton* m_downloadTemplateButton;
         TagContainerWidget* m_templateIncludedGems;
         QVector<ProjectTemplateInfo> m_templates;
+        QVector<TemplateButton*> m_templateButtons;
         FlowLayout* m_templateFlowLayout = nullptr;
         int m_selectedTemplateIndex = -1;
         bool m_userChangedProjectPath = false;
 
+        DownloadController* m_downloadController = nullptr;
```

Comment:
you can probably make this a QPointer. should offer you some level of safety if DownloadController is freed